package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "import-signing-certificate",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

```suggestion
  "version": "1.3.0",
```
This isn't really used, but should match the tags (1.2.0 currently, so next should be 1.3.0)